mirage/factories/utils.ts

```diff
@@ -114,3 +141,39 @@ export function createRegistrationMetadata(
     }
     return registrationMetadata;
 }
+
+/**
+ * Create registrationResponse according to schemaBlocks
+ */
+export function createRegistrationResponses(
+    registrationSchema: ModelInstance<MirageRegistrationSchema>,
+) {
+    const schemaBlocks = registrationSchema.schemaBlocks!;
```

Comment:
```suggestion
    const schemaBlocks = registrationSchema.schemaBlocks;
```
(may have to adjust type of `RegistrationSchema`)